src/components/pages/paymentPage/PaymentPageContainer.test.tsx

```diff
@@ -5,31 +5,27 @@ import { getPaymentPage } from './PaymentPageContainer';
 import { OrderStatus, OrderTypeEnum } from '../../../__generated__/globalTypes';
 
 describe('PaymentPageContainer', () => {
-  const t = jest.fn();
   it('should get correct page based on order status', () => {
```

Comment:
I think it's better to explicitly specify the number of assertions when the test implementation starts to get a bit complex.
```tsx
expect.assertions(6);
```

Reply:
Done.